cumulusci/tasks/robotframework/lint.py

```diff
@@ -0,0 +1,207 @@
+from cumulusci.core.tasks import BaseTask
+from cumulusci.core.utils import process_bool_arg
+from cumulusci.core.utils import process_list_arg
+from cumulusci.core.exceptions import CumulusCIFailure
+import rflint
+import glob
+import os
+
+
+class RobotLint(BaseTask):
+    task_docs = """
+    The robot_lint task performs static analysis on one or more .robot
+    and .resource files. Each line is parsed, and the result passed through
+    a series of rules. Rules can issue warnings or errors about each line.
+
+    If any errors are reported, the task will exit with a non-zero status.
+
+    When a rule has been violated, a line will appear on the output in
+    the following format:
+
+    *<severity>*: *<line>*, *<character>*: *<description>* (*<name>*)
+
+    - *<severity>* will be either W for warning or E for error
+    - *<line>* is the line number where the rule was triggered
+    - *<character>* is the character where the rule was triggered,
+      or 0 if the rule applies to the whole line
+    - *<description>* is a short description of the issue
+    - *<name>* is the name of the rule that raised the issue
+
+    Note: the rule name can be used with the ignore, warning, error,
+    and configure options.
+
+    Some rules are configurable, and can be configured with the
+    `configure` option. This option takes a list of values in the form
+    *<rule>*:*<value>*,*<rule>*:*<value>*,etc.  For example, to set
+    the line length for the LineTooLong rule you can use '-o configure
+    LineTooLong:80'. If a rule is configurable, it will show the
+    configuration options in the documentation for that rule
+
+    The filename will be printed once before any errors or warnings
+    for that file. The filename is preceeded by `+ `
```

Comment:
```suggestion
    for that file. The filename is preceeded by `+`
```

Without this change, running `cci task info robot_lint` prints a warning: `<string>:36: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.`

Reply:
Interesting. I should get in the habit of writing a test that verifies the task info command doesn't barf. That, or we create a test that iterates over all known tasks and runs the info command to make sure it doesn't barf so we don't have to remember to do it for each new task.